non-critical-infra/modules/jitsi.nix

```diff
@@ -0,0 +1,59 @@
+{ pkgs, ... }:
+{
+  services.jitsi-meet = {
+    enable = true;
+    hostName = "jitsi.nixos.org";
+    config = {
+      enableWelcomePage = true;
+      requireDisplayName = true;
+      analytics.disabled = true;
+      startAudioOnly = true;
+      channelLastN = 4;
+      lobby = {
+        autoKnock = true;
+        enableChat = false;
+      };
+      stunServers = [
```

Comment:
This should be `p2p.stunServers` according to the [documentation](https://jitsi.github.io/handbook/docs/devops-guide/turn) and [code](https://github.com/jitsi/jitsi-meet/blob/cc03949b90121350648503c6d2f8a8041f7245cc/config.js#L1005).

An alternative to using (and depending on) the matrix.org TURN server would be to disable the p2p calls for two person calls completely with `p2p.enabled = false`. One could argue this is a bit more privacy friendly, since this way you never expose your IP to the other person on the call.